aziotctl/src/internal/check/checks/mod.rs

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 
+#![allow(clippy::box_default)]
```

Comment:
Why not fix it?

Reply:
When I try to fix with the suggestions, I ran into the issue with `serde` complaining about `default` is not implemented.

Reply:
Nvm. I revisited the issue and was able to fix it. Not sure what stage of the code I was having to complain about serde.